alacritty/src/event.rs

```diff
@@ -91,8 +91,8 @@ pub struct SearchState {
     /// Change in display offset since the beginning of the search.
     display_offset_delta: isize,
 
-    /// Vi cursor position before search.
-    vi_cursor_point: Point,
+    /// Search origin in viewport coordinates relative to original display offset.
+    origin: Point,
```

Comment:
I've decided to rename this from the `vi_cursor_point` to just `origin` since it never really tracked the vi point to begin with. Even before this PR it was tracking the origin of the search which was unrelated to the vi cursor position. But since this PR changes search around anyways I've decided to include it here.